lua/telescope/path.lua

```diff
@@ -4,6 +4,15 @@ local path = {}
 path.separator = package.config:sub(1, 1)
 path.home = vim.fn.expand("~")
 
+path.exists = function(filepath)
+  return vim.loop.fs_stat(filepath) ~= nil
+end
```

Comment:
I think using `vim.loop.fs_stat` is not optimal because it will not only check if the file exists but collect almost everything about the file and returns it as a table.

Reply:
If `fs_stat` is calling `stat` that's what e.g. `os.path.exists` does in Python -- if you have a better suggestion though obviously I'm all ears.

Reply:
`fs_stat` just gathers and returns all [stats](https://github.com/luvit/luv/blob/master/docs.md#uvfs_statpath-callback), which i feel like is not optimal but it should really matter if we do that only on `_goto_file_selection`

Reply:
That says it's just calling `stat(2)`, so yeah at least as far as I know, that's as efficient as you can be.

Of course yeah creating all the Lua objects is useless work so you can FFI things to avoid that I guess but I doubt it matters much.